<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:reflect, x1:0, y1:1, x2:0.5, y2:0.5, stop:0 rgba(50, 50, 50, 200), stop:1 rgba(190, 255, 255, 255));</string>
  </property>
  <widget class="QLabel" name="Imagen">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>261</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="CargarImagenes">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>390</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50,50 , 50);
border-radius: 15px;
font: 75 13pt &quot;Helvetica&quot;;
color: white;</string>
   </property>
   <property name="text">
    <string>Cargar imagen</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="RutaArchivo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 75 12pt &quot;Helvetica&quot;;</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Browse">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>390</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50,50 , 50);
border-radius: 15px;
font: 75 13pt &quot;Helvetica&quot;;
color: white;</string>
   </property>
   <property name="text">
    <string>Buscar archivo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Corte">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>300</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
font: 75 12pt &quot;Helvetica&quot;;
</string>
   </property>
   <property name="text">
    <string>90</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="NivelMax">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>301</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 75 12pt &quot;Helvetica&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="NivelMax_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>300</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>12</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
font: 75 12pt &quot;Helvetica&quot;;
</string>
   </property>
   <property name="text">
    <string>Corte:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Avanzar">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>440</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50,50 , 50);
border-radius: 15px;
font: 75 13pt &quot;Helvetica&quot;;
color: white;
font-weight:bold</string>
   </property>
   <property name="text">
    <string>Siguiente</string>
   </property>
  </widget>
  <widget class="QLabel" name="NivelMax_3">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>300</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>12</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
font: 75 12pt &quot;Helvetica&quot;;</string>
   </property>
   <property name="text">
    <string>De</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="error">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>420</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
font: 10pt &quot;Helvetica&quot;;
color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="NivelMax_4">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>300</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>12</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
font: 75 12pt &quot;Helvetica&quot;;</string>
   </property>
   <property name="text">
    <string>capas</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Volver">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50,50 , 50);
border-radius: 15px;
font: 75 13pt &quot;Helvetica&quot;;
color: white;
font-weight:bold</string>
   </property>
   <property name="text">
    <string>Volver</string>
   </property>
  </widget>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>454</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
